### DISCUSSION TITLE
Coming Changes at Google Analytics require GA4

### DISCUSSION THREAD

**Discussion by dvshmkr** [rtp-book/code#6] [Errata]
Working through Chapter 24 now. I registered to obtain a tracking ID and was presented with this notice. Just an FYI since it appears to affect the chapter content.

_On July 1, 2023, this property will stop processing data. Starting in March 2023, for continued website measurement, you should create a new Google Analytics 4 (GA4) property, or one will be created for you based on your original property and reusing existing site tags._

**Comment by JennaSys** [comment]
Yea, that one there is no direct update for. Since the react-ga library is not being updated for compatibility with GA4, I've personally started using the [ga-gtag](https://www.npmjs.com/package/ga-gtag) JavaScript library instead. It's a little bit lower level and not quite as clean as using react-ga, but still pretty simple to use. I'll see if I can do a write-up on using it.

**Comment by JennaSys** [comment]
I outlined how to use what could be a possible [replacement for React-GA](https://github.com/rtp-book/code/discussions/10#discussion-6917362) in the Errata section that works with GA4.
